tests/e2e/forms/muac-follow-up.wdio-spec.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+const utils = require('../../utils');
+const userData = require('../../page-objects/forms/data/user.po.data');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+const genericForm = require('../../page-objects/forms/generic-form.wdio.page');
+const assessmentForm = require('../../page-objects/forms/assessment-form.wdio.page');
+const formTitle = assessmentForm.formDocument.title;
+
+describe('Assessment', () => {
+  before(async () => {
+    await assessmentForm.uploadForm();
+    userData.userContactDoc.date_of_birth = moment().subtract(4, 'months').format('YYYY-MM-DD');
+    await utils.seedTestData(userData.userContactDoc, userData.docs);
+    await loginPage.cookieLogin();
+    await commonPage.closeTour();
+    await (await commonPage.analyticsTab()).waitForDisplayed();
```

Comment:
I _think_ that we have a `commonPage.waitForPageLoaded` function. I think it would be clearer to use that instead of  explicitly waiting for a tab to be displayed.